zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGenerator.scala

```diff
@@ -31,12 +32,31 @@ class AQLGenerator extends IQLGenerator {
   //TODO combine with parseQuery
   override def calcResultSchema(query: Query, schema: Schema): Schema = {
     if (query.lookup.isEmpty && query.groups.isEmpty && query.select.isEmpty) {
-      schema.copy()
+      schema.copySchema
     } else {
       ???
     }
   }
 
+  private def splitSchemaMap(schemaMap: Map[String, AbstractSchema]): (Map[String, Schema], Map[String, LookupSchema]) = {
```

Comment:
I followed the logic of `calcResultSchema` to duplicate the code twice in both `AQLGenerator` and `AsterixQueryGenerator`. I didn't find a proper object to apply this static method. Which one you think maybe appropriate?